consensus/safety-rules/src/safety_rules.rs

```diff
@@ -458,3 +458,38 @@ where
             err
         })
 }
+
+#[cfg(any(test, feature = "fuzzing"))]
+/// This module allows the fuzzer to fuzz the guarded_* request handlers used by safety rules to
+/// handle message requests.
+pub mod fuzzing {
```

Comment:
Why can't this go into fuzz_utils or fuzzing.rs. This doesn't seem like the right place for it.

Reply:
As discussed offline, this was not avoid exposing the guarded_*_() methods. For now, I've updated PR to call the public interface methods and take the 20% performance hit for each fuzzing iteration!